src/pages/CartPage.js

```diff
@@ -308,64 +319,22 @@ const CartPage = () => {
                 })}
               </Box>
 
-              {deliveryMethod === 'delivery' && (
-                <Box sx={{ mt: 4 }}>
-                  {hasDeliveryDiscount || (cartTotal > amountForFreeDelivery && deliveryMethod === 'delivery') ? (
-                    <Box sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
-                      <Typography variant='h6' color='#031D44' sx={{ fontWeight: 'bold', mr: 1 }}>
-                        Taxa de entrega:
-                      </Typography>
-                      <Typography variant='body1' color='#031D44' sx={{ textDecoration: 'line-through', mr: 1 }}>
-                        {orderDeliveryFee}€
-                      </Typography>
-                      <Typography variant='body1' color='#031D44'>
-                        0€
-                      </Typography>
-                    </Box>
-                  ) : (
-                    <Box sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
-                      <Typography variant='h6' color='#031D44' sx={{ fontWeight: 'bold', mr: 1 }}>
-                        Taxa de entrega:
-                      </Typography>
-                      <Typography variant='body1' color='#031D44'>
-                        {orderDeliveryFee}€
-                      </Typography>
-                    </Box>
-                  )}
+              {deliveryMethod !== 'delivery' && (
+                <Box sx={{ display: 'flex', justifyContent: 'center', mt: 2 }}>
+                  <Typography variant='h4' color='#031D44' sx={{ fontWeight: 'bold', mr: 1 }}>
+                    Total:
+                  </Typography>
+                  <Typography variant='h4' color='#031D44'>
+                    {addedDeliveryFee && cartTotal < amountForFreeDelivery ? (cartTotal + orderDeliveryFee).toFixed(2) : cartTotal.toFixed(2)}€
```

Comment:
I would suggest to give names to these variable to make it easier to read and understand.

```suggestion
                    {shouldPayForDeliveryFee ? orderPriceWithFee : orderPrice}€
```